types/ui-select/index.d.ts

```diff
@@ -1,19 +1,77 @@
-// Type definitions for ui-select 0.13.2
+// Type definitions for ui-select 0.19.8
 // Project: https://github.com/angular-ui/ui-select
 // Definitions by: Niko Kovačič <https://github.com/nkovacic>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped 
+//                 Adam Kwiatek <https://github.com/akwiatek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
 /// <reference types="angular" />
 
 import * as angular from 'angular';
 
 declare module 'angular' {
-	export namespace ui.select {
-	    interface ISelectConfig {
-	        appendToBody: boolean;
-	        resetSearchInput: boolean;
-	        theme: string;
-	    }
-	}
+    export namespace ui.select {
+        interface ISelectConfig {
+            appendToBody: boolean;
+            backspaceReset: boolean;
+            closeOnSelect: boolean;
+            dropdownPosition: string;
+            generateId(): number;
+            paste?: IPasteFn;
+            placeholder: string;
+            refreshDelay: number;
+            removeSelected: boolean;
+            resetSearchInput: boolean;
+            searchEnabled: boolean;
+            skipFocusser: boolean;
+            sortable: boolean;
+            spinnerClass: string;
+            spinnerEnabled: boolean;
+            theme: string;
+        }
+
+        interface ISelectController {
+            activeIndex: number;
+            clickTriggeredSelect: boolean;
+            closeOnSelect: boolean;
+            close(skipFocusser?: boolean): void;
```

Comment:
`skipFocusser` is sent from
https://github.com/angular-ui/ui-select/blob/v0.19.8/src/uiSelectController.js#L443
to
https://github.com/angular-ui/ui-select/blob/v0.19.8/src/uiSelectSingleDirective.js#L75